src/domain/connect/base.ts

```diff
@@ -48,15 +50,27 @@ export const providers: Record<ProviderId, Provider> = {
     id: ProviderId.WalletConnect,
     name: 'WalletConnect',
   },
+  [ProviderId.Phantom]: {
+    id: ProviderId.Phantom,
+    name: 'Phantom',
+  },
 }
 
+export const defaultBlockchainProviders: Record<BlockchainId, ProviderId> = {
+  [BlockchainId.ETH]: ProviderId.WalletConnect,
+  [BlockchainId.AVAX]: ProviderId.WalletConnect,
+  [BlockchainId.BASE]: ProviderId.WalletConnect,
+  [BlockchainId.SOL]: ProviderId.Phantom,
+} as Record<BlockchainId, ProviderId>
+
 // ------------------------
 
 export type Blockchain = {
   id: BlockchainId
   name: string
   chainId: number
   eip155: boolean
+  solana: boolean
```

Comment:
This new field is not needed, we can use eip155: false https://namespaces.chainagnostic.org/eip155/caip2

Reply:
I thought on adding it as it is not harmful and will be easier to add new chains (e.g. polkadot) easily in the future